kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/CaseServicesClient.java

```diff
@@ -118,6 +121,8 @@
 
     List<CaseComment> getComments(String containerId, String caseId, Integer page, Integer pageSize);
 
+    List<CaseComment> getComments(String containerId, String caseId, String sortBy, Integer page, Integer pageSize);
```

Comment:
Should be better to have getComments(String containerId, String caseId, Integer page, Integer pageSize, String sortBy, boolean sortOrder)